diff --git a/old/node_modules/@tarp/require/require.js b/new/node_modules/@tarp/require/require.js
index 86136a22..f991aabe 100644
--- a/old/node_modules/@tarp/require/require.js
+++ b/new/node_modules/@tarp/require/require.js
@@ -25,7 +25,7 @@
   function resolve(id, pwd) {
     var matches = id.match(/^((\.)?.*\/|)(.[^.]*|)(\..*|)$/);
     return (new URL(
-      matches[1] + matches[3] + (matches[3] && (matches[4] || ".js")),
+      matches[1] + matches[3] + "/" + matches[3] + (matches[3] && (matches[4] || ".js")),
       pwd
     )).href;
   }
